tests/unit/disco/components/TestDiscoPane.js

```diff
@@ -335,6 +335,17 @@ describe(__filename, () => {
   });
 
   describe('See more add-ons link', () => {
+    it('shows a "more add-ons" link with UTM params', () => {
+      const root = render();
+      const button = root.find('.amo-link').find(Button);
+
+      expect(button).toHaveLength(1);
+      expect(button).toHaveProp(
+        'href',
+        'https://addons.mozilla.org/?utm_source=discovery.addons.mozilla.org&utm_medium=firefox-browser&utm_content=see-more-link&src=api',
```

Comment:
Should we be hardcoding this expectation, or should we be using the `makeQueryStringWithUTM` function to generate it?

Reply:
ahm, that's the thing I don't really know to be honest. I think, maybe I should use `makeQueryStringWithUTM()` yes, otherwise I am testing this function as well.